### DISCUSSION TITLE
Resize report errors

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#834] [Help]
![QQ截图20210108215043](https://user-images.githubusercontent.com/65677359/104023099-20460400-51fc-11eb-95be-853b171d2518.jpg)
In previous versions of resize, you can enter one item wide or high, and the other one is 0 to run
![1111](https://user-images.githubusercontent.com/65677359/104023221-4cfa1b80-51fc-11eb-8d72-5db4d8551f05.jpg)

**Comment by dlemstra** [comment]
Have you tried upgrading to the most recent version? Do you have a small code sample that reproduces this issue? And is your issue resolved when you do `OpenCL.IsEnabled=false`.

**Reply by jueqing1015** [reply]
use OpenCL.IsEnabled=false solve
